src/radio-tile/radio-tile.scss

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+@mixin iui-radio-tile {
+  cursor: pointer;
+  width: $iui-xl * 5;
+  height: 100%;
+  box-sizing: border-box;
+  padding: $iui-s;
+  position: relative;
+  z-index: 1;
+  border: 1px solid rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-4));
+  background-color: var(--iui-color-background-1);
+  @media (prefers-reduced-motion: no-preference) {
+    transition: border-color $iui-speed-fast ease-out;
+  }
+
+  // Rounded corners on first tile
+  &:first-of-type {
+    border-radius: $iui-border-radius 0 0 $iui-border-radius;
+  }
+
+  // Rounded corners on last tile
+  &:last-of-type {
+    border-radius: 0 $iui-border-radius $iui-border-radius 0;
+  }
+
+  // All tiles shift 1px left so we don't have 2px thick dividers
+  &:not(:first-of-type) {
+    margin-left: -1px;
+  }
+
+  &-container {
```

Comment:
Not sure I am a fan of this...
In other compnents we just had separate mixins for separate parts and then separate classes.
`.iui-radio-tile-container { @include iui-radio-tile-container }`
THis allows also for an easier search for that specific class.
Any reason you chose this approach?

Reply:
@veekeys do you mean you'd like it to align with [this sort of structure](https://github.com/iTwin/iTwinUI/blob/77480a0f6c0019b80724bef10821bd50eb9b2448/src/breadcrumbs/classes.scss)?  It wouldn't make it easier to search for a specific class, but it does keep the mixins smaller and easier to reuse.

Reply:
no, this is like that right now, almost.
Im tlaking more like this:
https://github.com/iTwin/iTwinUI/blob/main/src/date-picker/classes.scss
It does allow easy search for the specific class :)

Reply:
Ok, I'll update breadcrumbs to align, @adamhock i guess do the same to match.  Should be fairly easy.

Reply:
Mixins added